### PR TITLE
Update values.yaml example config to fix installation error

### LINKED ISSUE #217: [BUG] opensearch-dashboards helm install fails with 1.2.1
**Describe the bug**
Not able to install opensearch-dashboards using 1.2.1 version of helm chart.  My values file works fine with 1.2.0 and earlier.

```
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(ConfigMap.data.opensearch_dashboards.yml): invalid type for io.k8s.api.core.v1.ConfigMap.data: got "map", expected "string"
```

**To Reproduce**
Minimal values file (based on the default values) which reproduces the issue:
```
config:
  # Default OpenSearch Dashboards configuration from docker image of Dashboards
  opensearch_dashboards.yml:
    server:
      # name: dashboards
      # host: 0.0.0.0

      # Dashboards TLS Config (Ensure the cert files are present before enabling SSL
      ssl:
        enabled: true
        key: /usr/share/opensearch-dashboards/certs/dashboards-key.pem
        certificate: /usr/share/opensearch-dashboards/certs/dashboards-crt.pem
```

**Chart Name**
opensearch-dashboards

**Host/Environment (please complete the following information):**
 - Helm Version: 3.x
 - Kubernetes Version: 1.21

### PR DESCRIPTION
**Pull request by TheAlgo**
Signed-off-by: TheAlgo <dhirajjain0@gmail.com>

### Description
Update values.yaml example config to fix installation error. `|` symbol was missing
 
### Issues Resolved
#217 
 
### Check List
- [x] Commits are signed per the DCO using --signoff

For any changes to files within Helm chart directories:
- [x] Helm chart version bumped
- [x] Helm chart `CHANGELOG.md` updated to reflect change

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/helm-charts/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by TheAlgo** [comment]
@prudhvigodithi @peterzhuamazon have a look when free. Thanks

**Comment by prudhvigodithi** [comment]
this looks good @TheAlgo

File: charts/opensearch-dashboards/CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Update `values.yaml` example to use the correct format for configs
+### Changed
+### Deprecated
+### Removed
+### Fixed
+### Security
+---
 ## [1.4.0]
 ### Added
 - Change app version to use OpenSearch-Dashboards 1.3.1
```
Comment (peterzhuamazon) [inline charts/opensearch-dashboards/CHANGELOG.md:27]: Seems like spacing is not correct.

**Comment by TheAlgo** [comment]
> this looks good @TheAlgo, can you please check if [Unreleased] section change is missing in change-log file ?

It is present

**Comment by prudhvigodithi** [comment]
> > this looks good @TheAlgo, can you please check if [Unreleased] section change is missing in change-log file ?
> 
> It is present

So just checking this [PR](https://github.com/opensearch-project/helm-charts/pull/246/files#) there is an [Unreleased] section added.

**Comment by TheAlgo** [comment]
> > > this looks good @TheAlgo, can you please check if [Unreleased] section change is missing in change-log file ?
> > 
> > 
> > It is present
> 
> So just checking this [PR](https://github.com/opensearch-project/helm-charts/pull/246/files#) there is an [Unreleased] section added.

Ah I thought the upper section of `Changelog` my bad. From next time I will use my laptop for sending PR(s). 😅

**Review by prudhvigodithi — APPROVED** [review]

File: charts/opensearch-dashboards/CHANGELOG.md (hunk 1 above)
Comment (peterzhuamazon) [reply charts/opensearch-dashboards/CHANGELOG.md:27]: Seems like it is fixed now it used to show spacing are off by 1.

**Comment by sastorsl** [comment]
Similar to the discussion in #246 - can the title reflect the helm release version number?
It makes it easier to walk through the PR's and see what changed what.

This also touches into #173 regarding updated release notes.

Suggestion: `Update values.yaml example config to fix installation error. Bump helm chart to opensearch-dashboards-1.4.1`

**Pull request merged**